packaging/linux/docker/slim/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -e
+
+# If a command was passed and KEYBASE_SERVICE is not set, simply run the command
+if [ "$#" -gt 1 ] -a [[ -z "${KEYBASE_SERVICE}" ]]; then
+    exec "$@"
+    exit 0
+fi
+
+# If we're called with any args, don't pollute stdout and stderr and write the
+# logs into /var/log/keybase
+if [ "$#" -gt 1 ]; then
+    bash -c "keybase ${KEYBASE_SERVICE_ARGS:-"-debug"} service &> /var/log/keybase/service.log" &
```

Comment:
why does it shell out?
also if we redirect logs like that I don't think log sends will work - is that ok?

Reply:
i _think_ it should work (you'll get 2 log files) and i can change it in the future after we do some basic tests

i'll drop the shellout, it should probably work as-is without bash -c